src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -382,6 +382,7 @@
          This needs to be set after the SDK targets have been imported.
     -->
     <PrepareForBuildDependsOn>$(PrepareForBuildDependsOn);GetCFamilyFiles</PrepareForBuildDependsOn>
+    <LangVersion>latest</LangVersion>
```

Comment:
Could this cause the same sort of problem as we saw with the most recent version of sonar-dotnet i.e. using language features that aren't supported by the target runtime?